crates/nu-cli/src/commands/version.rs

```diff
@@ -42,15 +41,24 @@ impl WholeStreamCommand for Version {
 pub fn version(args: CommandArgs, _registry: &CommandRegistry) -> Result<OutputStream, ShellError> {
     let tag = args.call_info.args.span;
 
-    let mut indexmap = IndexMap::with_capacity(2);
+    let mut indexmap = IndexMap::with_capacity(3);
 
     indexmap.insert(
         "version".to_string(),
         UntaggedValue::string(clap::crate_version!()).into_value(&tag),
     );
+    let lgc = LastGitCommit::new()
+        .build()
+        .expect("Error instantiating last-git-commit");
```

Comment:
You could avoid a potential crash if git isn't available by doing this slightly different.

```rust
if let Ok(lgc) = LastGitCommit::new().build() {
  
    indexmap.insert(
        "short_commit_hash".to_string(),
        UntaggedValue::string(format!("{}", lgc.id().short())).into_value(&tag),
    );
    indexmap.insert(
        "long_commit_hash".to_string(),
        UntaggedValue::string(format!("{}", lgc.id().long())).into_value(&tag),
    );
}
```

Reply:
Updated